composer.json

```diff
@@ -6,8 +6,12 @@
     "homepage": "https://github.com/Flubit/merchant-api-client",
     "license": "MIT",
     "require": {
-        "php": ">=5.3.3",
-        "guzzle/guzzle": "~3.7"
+        "php": ">=5.4",
+        "guzzlehttp/guzzle": "~5.3"
```

Comment:
The thing is guzzle 3 is not supported anymore and the same goes for php 5.3.
I thought that could be a good idea update the supported versions.

The newest version of guzzle  is 6 and uses  php 5.5 :)